x/avs/types/genesis.go

```diff
@@ -5,17 +5,35 @@ package types
 // DefaultIndex is the default global index
 const DefaultIndex uint64 = 1
 
-// DefaultGenesis returns the default genesis state
-func DefaultGenesis() *GenesisState {
+func NewGenesisState(
+	avsInfos []AVSInfo,
+	taskInfos []TaskInfo,
+	blsPubKeys []BlsPubKeyInfo,
+	taskResultInfos []TaskResultInfo,
+	challengeInfos []ChallengeInfo,
+	taskNums []TaskID,
+	chainIDInfos []ChainIDInfo,
+) *GenesisState {
 	return &GenesisState{
-		Params: DefaultParams(),
+		AvsInfos:        avsInfos,
+		TaskInfos:       taskInfos,
+		BlsPubKeys:      blsPubKeys,
+		TaskResultInfos: taskResultInfos,
+		ChallengeInfos:  challengeInfos,
+		TaskNums:        taskNums,
+		ChainIdInfos:    chainIDInfos,
 	}
 }
 
+// DefaultGenesis returns the default genesis state
+func DefaultGenesis() *GenesisState {
+	return NewGenesisState(nil, nil, nil, nil, nil, nil, nil)
+}
+
 // Validate performs basic genesis state validation returning an error upon any
 // failure.
 func (gs GenesisState) Validate() error {
 	// this line is used by starport scaffolding # genesis/types/validate
 
-	return gs.Params.Validate()
+	return nil
```

Comment:
Please create this function to validate the contents of the genesis file. You can check the format of the address and ensure there are no duplicates, for example